docs/source/configure.rst

```diff
@@ -0,0 +1,507 @@
+.. py:module:: sopel.config.core_section
+
+================================
+The [core] configuration section
+================================
+
+A typical configuration file looks like this::
+
+    [core]
+    nick = Sopel
+    host = chat.freenode.net
+    use_ssl = false
+    port = 6667
```

Comment:
If you aren't planning to force-push this PR to do the `configure.rst` -> `configuration.rst` rename, ignore this. 😸

If you _will_ force-push one last time, could you update this example to give the SSL version of connecting to freenode? When users copy-and-paste this example (and they _will_), I want them to automatically get the secure version unless they specifically edit it to connect over plaintext. Sensible?